addon/components/ember-popper-base.js

```diff
@@ -246,30 +208,24 @@ export default Component.extend({
 
       // Store current values to check against on updates
       this._didRenderInPlace = renderInPlace;
-      this._eventsEnabled = eventsEnabled;
       this._modifiers = modifiers;
-      this._onCreate = onCreate;
-      this._onUpdate = onUpdate;
+      this._onFirstUpdate = onFirstUpdate;
       this._placement = placement;
+      this._strategy = strategy;
       this._popperTarget = popperTarget;
 
       const options = {
-        eventsEnabled,
         modifiers,
-        placement
+        placement,
+        strategy
       };
 
-      if (onCreate) {
-        assert('onCreate of ember-popper must be a function', typeof onCreate === 'function');
-        options.onCreate = onCreate;
-      }
-
-      if (onUpdate) {
-        assert('onUpdate of ember-popper must be a function', typeof onUpdate === 'function');
-        options.onUpdate = onUpdate;
+      if (onFirstUpdate) {
+        assert('onFirstUpdate of ember-popper must be a function', typeof onFirstUpdate === 'function');
+        options.onFirstUpdate = onFirstUpdate;
       }
 
-      this._popper = new Popper(popperTarget, this._popperElement, options);
+      this._popper = Popper.createPopper(popperTarget, this._popperElement, options);
```

Comment:
I am a bit newbie here so I am interested why can't we import popper like `import { createPopper } from '@popperjs/core';` and have to use the UMD way? 🤔